glean-core/src/upload/directory.rs

```diff
@@ -76,41 +95,49 @@ impl PingDirectoryManager {
     pub fn new<P: Into<PathBuf>>(data_path: P) -> Self {
         let data_path = data_path.into();
         Self {
-            pings_dirs: [
-                data_path.join(PENDING_PINGS_DIRECTORY),
-                data_path.join(DELETION_REQUEST_PINGS_DIRECTORY),
-            ],
+            pending_pings_dir: data_path.join(PENDING_PINGS_DIRECTORY),
+            deletion_request_pings_dir: data_path.join(DELETION_REQUEST_PINGS_DIRECTORY),
         }
     }
 
     /// Attempts to delete a ping file.
     ///
-    /// ## Arguments
+    /// # Arguments
     ///
     /// * `uuid` - The UUID of the ping file to be deleted
     ///
-    /// ## Panics
+    /// # Return value
+    ///
+    /// `bool` - `true` if file was deleted succesfully and `false` otherwise.
```

Comment:
No need to mention the type again, that's in the function declaration.
```suggestion
    /// Returns `true` if the file was deleted succesfully and `false` otherwise.
```